src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1278,82 +1254,77 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                         $myinv = $err.InvocationInfo
                                     }
 
-                                    if ($ErrorView -ne 'ConciseView' -and $null -ne $host.PrivateData) {
-                                        if ($null -ne $host.PrivateData.ErrorForegroundColor) {
-                                            [Console]::ForegroundColor = $host.PrivateData.ErrorForegroundColor
-                                        }
+                                    if ($err.FullyQualifiedErrorId -eq 'NativeCommandErrorMessage' -or $err.FullyQualifiedErrorId -eq 'NativeCommandError') {
+                                        return ""${errorColor}$($err.Exception.Message)${resetcolor}""
+                                    }
 
-                                        if ($null -ne $host.PrivateData.ErrorBackgroundColor) {
-                                            [Console]::BackgroundColor = $host.PrivateData.ErrorBackgroundColor
-                                        }
+                                    $myinv = $err.InvocationInfo
+                                    if ($ErrorView -eq 'DetailedView') {
+                                        $message = Get-Error | Out-String
+                                        return ""${errorColor}${message}${resetcolor}""
```

Comment:
In this case, I think `Get-Error` would have already done it's own error coloring so just `$message` should be sufficient.  Maybe add a comment?

Reply:
- When `$PSStyle.OutputRendering` is `Host`, which is the default value, or `PlainText`, `Get-Error | Out-String` will have all ANSI decorations striped off, and hence no color will be applied to the detailed error rendering without adding `$errorColor`.
- When `PSStyle.OutputRendering` is `Ansi`, then the added `$errorColor` will be overridden by the colors from `Get-Error`.

It's a little inconsistent here depending on the value of `OutputRendering`. If we really want it to be consistent, then we can set `$PSStyle.OutputRendering` to either `Host` or `Ansi` (depending on whether we want it to be rendered as error color, or the original color from `Get-Error`), and then revert `OutputRendering` to its original value. Personally, I think this inconsistency is not a big deal; what's your thoughts?

Reply:
I see.  Perhaps for now what you have is fine.  The extra characters aren't a huge deal.